.circleci/config.yml

```diff
@@ -33,6 +36,17 @@ jobs:
       - store_test_results:
           path: junit-reports
       - run: yarn run codecov
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+  snyk:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace: { at: . }
+      - run: yarn run snyk test --severity-threshold=high --org=segment-pro
```

Comment:
When using the snyk auth token that gets pulled in by `context: snyk` you shouldn't need to specify a --org. If you remove this and look at the tests you should still see `Organisation:  segment-pro` in the snyk test and snyk monitor in CI.

Reply:
updated!